scripts/pulling_sim4life_images/template.env

```diff
@@ -0,0 +1,4 @@
+GITLAB_ACCESS_TOKEN=
```

Comment:
Suggestion:
Do `GITLAB_ACCESS_TOKEN=${GITLAB_ACCESS_TOKEN=}` and add the vars to `repo.config` :)